src/inverseRobot.js

```diff
@@ -4,10 +4,22 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const res = {};
```

Comment:
3. [Checklist] - use proper names for variables - if you want to name variable result, pick a better one :)